src/utils/Table/TableContent.js

```diff
@@ -67,7 +67,7 @@ TableRow.propTypes = {
     rowProps: PropTypes.object.isRequired,
     handleSelect: PropTypes.func.isRequired,
     isRowSelected: PropTypes.func.isRequired,
-    RowActions: PropTypes.elementType.isRequired,
+    RowActions: PropTypes.elementType,
```

Comment:
I changed this since we could have tables without RowActions. Is that alright? or should we enforce it?

Reply:
For that case you can remove it, yes.